frontend/src/components/elements/CodeBlock/CodeBlock.tsx

```diff
@@ -54,9 +55,11 @@ export default function CodeBlock({
   if (language == null) {
     return (
       <StyledCodeBlock className="stCodeBlock">
-        <StyledCopyButtonContainer>
-          <CopyButton text={value} />
-        </StyledCopyButtonContainer>
+        {!_.isEmpty(value) && (
+          <StyledCopyButtonContainer>
+            <CopyButton text={value} />
+          </StyledCopyButtonContainer>
+        )}
         <StyledPre>
           <code>{value}</code>
```

Comment:
Hm, so this isn't really originally in the scope of your change, so feel free to ignore.

I think it'd be a good idea to lightly refactor this since everything aside from the contents of the `code` block is identical between the two cases. Maybe doing something like writing 
```javascript
const renderCode = () => {
    if (lang === null) {
        return <code>{value}</code>
    } else {
        // lang processing stuff
        return <code className={languageClassName} dangerouslySetInnerHTML={{ __html: safeHtml }} />=
    }
}
```
to call within the `<StyledPre>` can get rid of the duplication here

Reply:
Please create a separate PR for that.